src/ctoken.ts

```diff
@@ -360,6 +391,12 @@ export function handleTransfer(event: Transfer): void {
     ])
   } else {
     if (balanceOfToAccountResult.value.equals(event.params.amount)) {
```

Comment:
When handling `Transfer` events, if the resulting balance of the `to` account is the same as the `amount` of the transfer, that indicates the starting balance of the `to` account was 0, so they are technically a new supplier to that market (by virtue of simply holding the cTokens). If true, this will increment `supplierCount`, irrespective of the balance of the `source` account.